spyder/utils/introspection/plugin_client.py

```diff
@@ -77,7 +81,7 @@ def run(self):
         # Set up environment variables.
         processEnvironment = QProcessEnvironment()
         env = self.process.systemEnvironment()
-        if (self.env and 'PYTHONPATH' not in self.env) or DEV:
+        if (self.env and 'PYTHONPATH' not in self.env) or self.env is None:
```

Comment:
Please restore `DEV` here. I think it's needed to make completions work when Spyder is started with `bootstrap.py`